src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -134,6 +135,11 @@ public async Task<ITriggerData> BindAsync(object value, ValueBindingContext cont
                     throw new NotSupportedException("An HttpRequest is required");
                 }
 
+                // apply some invocation details to the request that can be used later
+                // in the http request pipeline
+                request.HttpContext.Items[ScopeKeys.FunctionInvocationId] = context.FunctionInstanceId.ToString();
+                request.HttpContext.Items[ScopeKeys.FunctionName] = context.FunctionContext.MethodName;
```

Comment:
I'm going to use these properties to allow me to remove 2 of the 4 ETW invocation events we emit on each http function invocation. For http, we have trace middleware in the pipeline that writes detailed logs for all http requests, so the standard WebJobs Executing/Executed are unnecessary duplication. By omitting them, I'm seeing a 5% RPS gain in load tests, since we're effectively reducing ETW events by 50%.

I need these 2 bits of info to ensure the middleware logs have all the required details - they're missing name/id currently.

Reply:
This may be problematic. I'm pretty sure we have detectors that rely on those Executing/Executed events and we've added some sample queries in Log Analytics that rely on them as well. We'd need a good way to see all your executions (HTTP or other) in a query and try to keep it as simple as possible.

Reply:
I've removed this change from the PR, until we decide on it.